providers/azure/resources/azure.lr

```diff
@@ -1183,7 +1183,7 @@ private azure.subscription.sqlService.database @defaults("id name") {
   readScale string
   // SQL database sample name
   sampleName string
-  // Whether SQL server is zone redundant
+  // Whether the SQL is zone redundant
```

Comment:
really, just the SQL?

Reply:
This sounds like the query language is zone redundant.

Reply:
maybe we could make this database instead